From 102e28ebb57b7d42f1ac8dbd361bef372ccb2106 Mon Sep 17 00:00:00 2001
Message-Id: <102e28ebb57b7d42f1ac8dbd361bef372ccb2106.1413837857.git.chang-joon.lee@intel.com>
In-Reply-To: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
References: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
From: Gajanan Bhat <gajanan.bhat@intel.com>
Date: Wed, 6 Aug 2014 20:43:06 +0530
Subject: [PATCH 242/312] FOR_UPSTREAM [VPG]: drm/i915: Enable sprite flip
 notification for CHV

For CHV enable sprite flip notification bits in PIPESTAT. With this
sprite flip done interrupts are getting generated properly.

Issue: GMIN-3128
Change-Id: Iaba9cc9a5b71f40ae974e13fd769a20924bb0501
Signed-off-by: Gajanan Bhat <gajanan.bhat@intel.com>
---
 drivers/gpu/drm/i915/i915_irq.c |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/gpu/drm/i915/i915_irq.c b/drivers/gpu/drm/i915/i915_irq.c
index 9db7673..ca2e094 100644
--- a/drivers/gpu/drm/i915/i915_irq.c
+++ b/drivers/gpu/drm/i915/i915_irq.c
@@ -3874,6 +3874,8 @@ static int cherryview_irq_postinstall(struct drm_device *dev)
 		I915_DISPLAY_PIPE_B_EVENT_INTERRUPT |
 		I915_DISPLAY_PIPE_C_EVENT_INTERRUPT;
 	u32 pipestat_enable = PLANE_FLIP_DONE_INT_STATUS_VLV |
+		SPRITE0_FLIP_DONE_INT_STATUS_VLV |
+		SPRITE1_FLIP_DONE_INT_STATUS_VLV |
 		PIPE_CRC_DONE_INTERRUPT_STATUS |
 		PIPE_DPST_EVENT_STATUS;
 	unsigned long irqflags;
-- 
1.7.9.5

